### ISSUE TITLE
Cannot start eclipse plugin

### ISSUE THREAD

**Issue by GoogleCodeExporter** [umlet/umlet#219]
```
What steps will reproduce the problem?
1. Copy com.umlet.plugin_13.1.0.jar to plugins/
2. Start eclipse

What is the expected output? What do you see instead?
Expected: I expected to be able to create a new umlet diagram by choosing File 
> New > Other > UMLet (or something like that)
Instead: There is no option for creating a new UMLet diagram and if I double 
click on a uxf-file, it will just show the xml (and I cannot find the UMLet 
editor from the open-with menu).

What version of the product are you using? On what operating system?
Eclipse Luna, UMLet plugin 13.1.0, Windows 7, Java 1.8.0_25

Please provide any additional information below.
The stand-alone version of 13.1.0 works.
If I do the same with com.umlet.plugin_12.2.0.jar it works...
```

Original issue reported on code.google.com by `t...@tueh.dk` on 10 Dec 2014 at 1:19

**Comment by Brian-Newport** [comment]
I have the same problem with UMLet 13.2.0 in Luna Service Release 1 (4.4.1) on Mac OS-X  10.9.5. No problem when reverted to UMLet 12.2.0, so the problem seems to have been introduced in UMLet 13.

Specifically with 13.2.0 the UMLet feature is not added to any eclipse UI element, including file associations. Nothing related to UMLet in workspace/.metadata/.log.

I did 'ls -lu' on the plugins directory, and see that the UMLet jar file was read at the same time as the others.

**Comment by afdia** [comment]
Does it work with v13.0?

**Comment by Brian-Newport** [comment]
Not with v13.0 either - just tried that.

**Comment by afdia** [comment]
Although I haven't found the issue (as I cannot reproduce it unfortunately), I have found several places where Exceptions have been caught and logged with log4j (which doesnt output anything in Eclipse)

I have changed this for the next release, so at least you should get an Exception which is visible in the logs. From there on it should be possible to fix the issue hopefully.

**Comment by afdia** [comment]
Fortunately we've found the issue and fixed it in #306

This only happens on Eclipse version without jdt.core component.

It can be checked by:
a. Go to console view
b. from the dropdown select "Host OSGi Console"
c. enter "ss" and confirm
d. if there is no entry like "248 ACTIVE org.eclipse.jdt.core_3.11.0.v20150602-1242", v13.0 to v13.3 won't work because if the missing dependency

I've set the dependency to optional and checked if it works in Eclipse C/C++ (which has no jdt.core) and it does therefore this should be fixed
